ztp/siteconfig-generator/siteConfig/siteConfigBuilder_test.go

```diff
@@ -545,6 +613,15 @@ spec:
 	}
 }
 
+func Test_SNOClusterSiteConfigBuildWithoutNetworkType(t *testing.T) {
```

Comment:
@imiller0 I have added this test  where networkType wasn't set

Reply:
Ah, ok and siteConfigTestWithoutNetworkType includes an installConfigOverride. Perfect, thank you!